### DISCUSSION TITLE
search results change from ver 3.3.1 to 3.5.6 (questions)

### DISCUSSION THREAD

**Discussion by kavren** [netbox-community/netbox#13384] [Q&A] [answered]
In the netbox rev 3.3.1, when I would do a search from the main search bar on the home page, the search results gave a lot of information. I can't remember it exactly, but I know that when I put in the first two octets of an ip in to search, I would get a quick result here I could pull vlan info, and I think comments, and just a lot of good info.

In the current rev that I am on, 3.5.6, a lot of that detailed info is missing when I do a search from the main search bar on the home page. When I do a search from the prefixes section, in the quick search bar, I get a lot more data returned.

So with all the above said: is there a way I can sculpt the main search results from the home-page search bar, to add in more columns and fields (to give details like from the 3.3.1 rev)?

Thank you for your time!

**Comment by abhi1693** [comment, marked as answer]
You should see https://github.com/netbox-community/netbox/discussions/11665 and https://github.com/netbox-community/netbox/discussions/12314

**Reply by kavren** [reply]
Thank you!
